Update user-input xsd to validate omitOnAuto (IZPACK-1138)

LevinGermann opened the pull request:
The xsd has been updated to validate the attribute "omitOnAuto".
It might have been forgotten in:
https://izpack.atlassian.net/browse/IZPACK-1138

LevinGermann commented:
Please note that I found out that the [documentation ](https://izpack.atlassian.net/wiki/spaces/IZPACK/pages/491620/Fields) is slightly wrong.
It states the following:
> The <field> tag has some common attributes: [...]
But the "omitFromAuto" attribute seems to be set on the <spec> element, not on the <field> ...

rkrell commented:
> Please note that I found out that the documentation is slightly wrong.
You can edit or comment the documentation directly if you find something wrong.
Nevertheless, your comment ends somewhere in the middle, I did not get it completely.

LevinGermann commented:
Thanks for responding on this PR! :)
I changed the documentation and updated the type to be boolean. It was a boolean before but I think I mixed something up in the next commit.

rkrell commented:
Thanks for contributing.
